### PR TITLE
Fixing issues with `spike-contentful` documentation

### PR DESCRIPTION
**Pull request by mirshko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -221,7 +221,6 @@ new Contentful({
       name: 'press',
       id: '4Em9bQeIQxxxxxxxxx'
       // No JSON output needed for this content type
-    }
```
Comment (jescalan) [inline README.md]: I think this close bracket is needed...

Comment (mirshko) [inline README.md]: Oh weird. I totally didn't think that came with the commit. Ill fix.

**Comment by mirshko** [comment]
@jescalan Should be good now!

**Comment by jescalan** [comment]
Perfect! Thanks for catching and correcting this ✨

**Pull request merged**